packages/broker/test/integration/per-node-metrics.test.ts

```diff
@@ -48,9 +50,40 @@ const fillMetrics = async (client: StreamrClient, count: number, nodeAddress: st
     return Promise.allSettled(promises)
 }
 
+const waitForMessage = (
+    stream: string,
+    client: StreamrClient, 
+): Promise<Todo> => {
+    return new Promise((resolve, reject) => {
+        try {
+            client.subscribe({stream}, (res: Todo) => {
+                resolve(res)
+            })
```

Comment:
Unfortunately `any` is the type that streamr-client gives us for `subscribe` cb function parameter :shrug: 
```suggestion
            client.subscribe({stream}, (res: any) => {
                resolve(res)
            })
```